client/gui_rpc_server_ops.cpp

```diff
@@ -1650,10 +1649,17 @@ static bool is_http_post_request(char* buf) {
     if (!p) return false;
     p += 4;
     if ((int)strlen(p) < n) return false;
-    strcpy_overlap(buf, p);
     return true;
 }
 
+// remove HTTP header from request
+//
+static void strip_http_header(char* buf) {
+    char* p = strstr(buf, "\r\n\r\n");
+    p += 4;
```

Comment:
Dangerous arithmetic: if substring is not found, return value will be NULL, and this code will lead to segfault

Reply:
This is called only if is_http_post_request() return true,
so we already know the string is there.